README.md

```diff
@@ -107,6 +108,7 @@ public class SimpleModel {
 ### Step 3: Start Validating
 
 ```java
+
```

Comment:
This isn't necessary, remove it. Same with the one added below

Reply:
ok